service/openapi.yaml

```diff
@@ -204,8 +163,6 @@ components:
     evaluationRequest:
       description: Flag evaluation request
       properties:
-        flagType:
-          $ref: '#/components/schemas/flagType'
         context:
           $ref: '#/components/schemas/context'
     success:
```

Comment:
Rename to `evaluationSuccess` to be consistent with `bulkEvaluationSuccess`. 

Also, should we just include the same `errorCode` and `errorDetails` params here and just make the `flags` in `bulkEvaluationSuccess` use this shared schema?

Reply:
also rename the `failure` schema

Reply:
I’ve actually made that change in the other PR